packages/dapp-high-roller/src/components/app-provider/app-provider.tsx

```diff
@@ -119,6 +120,18 @@ export class AppProvider {
       state.playerSecondNumber
     );
 
+    if (state.stage === HighRollerStage.REVEALING) {
+      // TODO randomize this and save it in proposingPlayer state
+      const numberSalt =
+        "0xdfdaa4d168f0be935a1e1d12b555995bc5ea67bd33fce1bc5be0a1e0a381fc90";
```

Comment:
To randomize just to `ethers.utils.hexlify(ethers.utils.randomBytes(32))`

Reply:
Yea I didn't have the time to include it in this PR. Can we create an issue for this to be fixed tomorrow?